app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RatingsController < ApplicationController
+  
+
+  def show
+  end
+
+  def index
+    @ratings = Rating.all
```

Comment:
Should this be scoped by the `current_user`?

Reply:
Interesting - you are right, so I am trying to figure out how the output in the `ratings/index` page returns the correctly scoped data.

I am using `current_user` in the `ratings/index` page, I'm wondering how it's not returning an error since it's not defined in the Controller?